tests/integration/root.bats

```diff
@@ -11,7 +11,7 @@ function setup() {
 function teardown() {
 	if [ -n "$ALT_ROOT" ]; then
 		ROOT=$ALT_ROOT __runc delete -f test_dotbox
-		rm -rf "$ALT_ROOT"
+		unset ALT_ROOT
```

Comment:
Was this `rm -rf` not doing something meaningful/useful?

Reply:
If you look at `setup()`, `$ALT_ROOT` is a subdirectory of `$ROOT`, which is removed by `teardown_bundle`, so this extra `rm -rf` is not really needed.

I'll rework this part.

Reply:
Reworked ALT_ROOT handling in `teardown` is a separate commit; PTAL.